pyproject.toml

```diff
@@ -40,7 +40,7 @@ werkzeug = "2.0.3" # Flask breaks with later versions of Werkzeug
 filelock = "^3.0.12"
 mypy = "^0.790"
 pytest = "^6.2.4"
-pytest-cov = "^2.12.1"
+pytest-cov = "^3.0.0"
```

Comment:
[This PR ](https://github.com/votingworks/arlo/pull/1997)should have fixed the issue that was leading to the `coverage.Tracer` error, so this shouldn't be needed anymore if rebased off of main. Might be worth undoing this change and checking if it works as expected, though this change might be harmless